02_assignments/assignment_2.ipynb

```diff
@@ -96,10 +96,30 @@
       },
       "outputs": [],
       "source": [
-        "with open(all_paths[0], 'r') as f:\n",
+        "all_paths = [\n",
+        "  \"../05_data/assignment_2_data/inflammation_01.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_02.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_03.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_04.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_05.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_06.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_07.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_08.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_09.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_10.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_11.csv\",\n",
+        "  \"../05_data/assignment_2_data/inflammation_12.csv\",]\n",
+        "\n",
+        "with open (all_paths [0], 'r') as f:\n",
         "    # YOUR CODE HERE: Use the readline() method to read the .csv file into 'contents'\n",
-        "    \n",
-        "    # YOUR CODE HERE: Iterate through 'contents' using a for loop and print each row for inspection"
+        "    contents = f.readlines()\n",
+        "    for i in contents:\n",
+        "       print (i)\n",
+        "        \n",
+        "    # YOUR CODE HERE: Iterate through 'contents' using a for loop and print each row for inspection\n",
+        "  \n",
+        "    for row in contents:\n",
+        "     print (row)"
```

Comment:
Why iterating the same thing twice?

Reply:
"all_paths = [\n",
         "  \"../05_data/assignment_2_data/inflammation_01.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_02.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_03.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_04.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_05.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_06.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_07.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_08.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_09.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_10.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_11.csv\",\n",
         "  \"../05_data/assignment_2_data/inflammation_12.csv\",]\n",
         "\n",
         "with open (all_paths [0], 'r') as f:\n",
contents = f.readlines()\n",
         "    for i in contents:\n",
         "       print (i)\n",